assets/src/support-page/index.js

```diff
@@ -0,0 +1,48 @@
+/* global ampSupportData */
```

Comment:
We could use the same approach to using external data as in other entry points, i.e.:
- add an `externals` entry in the `webpack.config.js` (e.g. like in [the case of the Onboarding Wizard](https://github.com/ampproject/amp-wp/blob/2df1d2a7085883a6a7f6cfba0729d8b06ffba1b5/webpack.config.js#L237-L239)).
- import the external data as if it was a package (continuing on the [Onboarding Wizard example](https://github.com/ampproject/amp-wp/blob/2df1d2a7085883a6a7f6cfba0729d8b06ffba1b5/assets/src/onboarding-wizard/index.js#L11-L21)).

Reply:
Have update webpack config.